pkg/kando/location.go

```diff
@@ -38,17 +50,94 @@ func newLocationCommand() *cobra.Command {
 	cmd.AddCommand(newLocationDeleteCommand())
 	cmd.PersistentFlags().StringP(pathFlagName, "s", "", "Specify a path suffix (optional)")
 	cmd.PersistentFlags().StringP(profileFlagName, "p", "", "Pass a Profile as a JSON string (required)")
-	_ = cmd.MarkFlagRequired(profileFlagName)
+	cmd.PersistentFlags().StringP(repositoryServerFlagName, "r", "", "Pass a Repository Server CR as a JSON string (required for kopia based blueprints)")
 	return cmd
 }
 
+// NewDataMover creates an instance of DataMover Interface and returns
+// the preferred DataMover as per the arguments passed in kando command
+func NewDataMover(dm *DataMoverArgs, outputName, snapJson string) datamover.DataMover {
+	switch dm.Type {
+	case DataMoverTypeProfile:
+		return &datamover.Profile{
+			OutputName: outputName,
+			Profile:    dm.Profile,
+			SnapJSON:   snapJson,
+		}
```

Comment:
I think it's better here in `pkg/kando/location.go` because in `pkg/datamover/` we're just having the datamover interface implementation. I'd love some inputs from others as well.
CC @PrasadG193 @viveksinghggits